frontend/src/components/LandingPage/LandingPage.module.css

```diff
@@ -0,0 +1,204 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  font-family: 'Lucida Sans', Verdana, sans-serif;
+  background-color: #f0f0f0;
+}
+
+h1 {
+  text-align: center;
+  color: #38023b;
+  padding-top: 20px;
+}

```

Comment:
This h1 style will be applied to all h1 tags in your codebase, so if you want to locally scope this style, use a classname instead of an h1 tag.